pkg/cmd/server/api/v1/types.go

```diff
@@ -626,3 +637,16 @@ type PodManifestConfig struct {
 	// The interval needs to be a positive value
 	FileCheckIntervalSeconds int64 `json:"fileCheckIntervalSeconds"`
 }
+
+type ExtensionFilesConfig struct {
+	// SubContext is the path under /<context>/extensions/ to serve files from Location
+	SubContext string `json:"subcontext"`
+	// Location is a directory on the asset server to serve files under SubContext in the
+	// Web Console. It may have nested folders.
+	Location string `json:"location"`
+	// HTML5Mode determines whether to redirect to the root index.html when a file is not found.
+	// This is needed for apps that use the HTML5 history API like AngularJS apps with HTML5
+	// mode enabled. If HTML5Mode is true, also rewrite the base element in index.html with the
+	// Web Console's context root. Defaults to false.
+	HTML5Mode bool `json:"html5Mode"`
```

Comment:
Yeah, I originally had it as two. We only inject base if index.html contains exactly `<base href="/">`, so if you only want the to redirect to index.html, it's possible.

I could see apps wanting to inject the base, but not redirecting to index.html. Hm.